### ISSUE TITLE
keadm didn't install the right version of cloudcore even though I specified --kubeedge-version flag

### ISSUE THREAD

**Issue by James-JLC** [kubeedge/kubeedge#4388]
keadm has been updated to version 1.12.1, see below
`$keadm version`
`$version: version.Info{Major:"1", Minor:"12", GitVersion:"v1.12.1", GitCommit:"919ad5378eaca3cb0c666c22a19db01261cbc9a6", GitTreeState:"clean", BuildDate:"2022-11-02T08:22:56Z", GoVersion:"go1.17", Compiler:"gc", Platform:"linux/amd64"}`

I tried install cloudcore of v1.12.1 with the following command
`keadm init --advertise-address=xxx.xxx.xxx.xxx --kubeedge-version=1.12.1 --kube-config=/home/anyone/.kube/config`

But v1.12.0 was installed successfully.

```W1108 10:43:39.788375       1 validation.go:154] TLSTunnelPrivateKeyFile does not exist in /etc/kubeedge/certs/server.key, will load from secret
W1108 10:43:39.788435       1 validation.go:157] TLSTunnelCertFile does not exist in /etc/kubeedge/certs/server.crt, will load from secret
W1108 10:43:39.788447       1 validation.go:160] TLSTunnelCAFile does not exist in /etc/kubeedge/ca/rootCA.crt, will load from secret
I1108 10:43:39.788468       1 server.go:92] Version: v1.12.0
W1108 10:43:39.788488       1 client_config.go:615] Neither --kubeconfig nor --master was specified.  Using the inClusterConfig.  This might not work.
```

**Comment by gy95** [comment]
you can try 
`keadm init --advertise-address=xxx.xxx.xxx.xxx --profile version=v1.12.1 --kube-config=/home/anyone/.kube/config`
with `--profile version=v1.12.1` but not `--kubeedge-version=1.12.1`

**Comment by James-JLC** [comment]
> you can try `keadm init --advertise-address=xxx.xxx.xxx.xxx --profile version=v1.12.1 --kube-config=/home/anyone/.kube/config` with `--profile version=v1.12.1` but not `--kubeedge-version=1.12.1`

Yes, I know. But it's not the right way to do that because we have kubeedge-version flag.

**Comment by James-JLC** [comment]
And if `--profile version=v1.12.1` is recommended, then we should remove `--kubeedge-version` flag, because it's confused and useless.

**Comment by James-JLC** [comment]
Here is another issue, When I run command `sudo keadm init --advertise-address='xxx.xxx.xxx.xxx' --profile=iptablesmgr=external --kube-config=/home/ubuntu/.kube/config` to install cloudcore and iptablesManager. It will try to pull `kubeedge/iptables-manager:1.12.0` and `kubeedge/cloudcore:1.12.0` by default, prefix **v** is missing.

I created a new [PR](https://github.com/kubeedge/kubeedge/pull/4393) to fix it.
